docs/sidebar.yml

```diff
@@ -166,11 +166,21 @@ subsection:
     directory: docs/contributing
     index: contributing/index.md
     subsection:
-      - page: contributing/contribute-knowledge.md
       - page: contributing/getting-started.md
       - page: contributing/workflow.md
-      - page: contributing/testing.md
-      - page: contributing/debugging.md
+      - title: Steps to Contribute
```

Comment:
Instead of having this "Steps to Contribute" what do you think about just adding all the children from this into the `workflow` section and make that contain all the children?